src/sql/azdata.proposed.d.ts

```diff
@@ -1238,7 +1238,7 @@ declare module 'azdata' {
 			indexColumnSpecificationTableOptions?: TableDesignerBuiltInTableViewOptions;
 		}
 
-		export interface TableDesignerBuiltInTableViewOptions {
+		export interface TableDesignerBuiltInTableViewOptions extends DesignerTablePropertiesBase {
```

Comment:
Any reason for having most of the classes be `TableDesigner...` but the new one be `DesignerTable...`?

Reply:
:) the newly added interface is table component properties base interface for designer.